packages/rfw/lib/src/flutter/core_widgets.dart

```diff
@@ -193,6 +194,9 @@ import 'runtime.dart';
 ///    to a [Rect] given by the `rect` key and a [TextDirection] given by the
 ///    `textDirection` key. New shader types can be added using
 ///    [ArgumentDecoders.shaderDecoders].
+/// 
+///  * the [SliverPersistentHeaderDelegate] type is represented as a map with a
+///    `type` key that can be `fixed` for [FixedSliverPersistentHeaderDelegate].
```

Comment:
more detail here would be nice

Reply:
(see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#writing-prompts-for-good-documentation)